extensions/schema-compare/src/dialogs/schemaCompareDialog.ts

```diff
@@ -610,16 +602,21 @@ export class SchemaCompareDialog {
 			this.dialog.okButton.enabled = await this.shouldEnableOkayButton();
 		});
 
-		// if target is currently a db, show it in the server and db dropdowns
-		if (this.schemaCompareMainWindow.targetEndpointInfo && this.schemaCompareMainWindow.targetEndpointInfo.endpointType === mssql.SchemaCompareEndpointType.Database) {
+		if (this.schemaCompareMainWindow.targetEndpointInfo) {
+			if (this.schemaCompareMainWindow.targetEndpointInfo.endpointType === mssql.SchemaCompareEndpointType.Database) {
+				databaseRadioButton.checked = true;
```

Comment:
can you also update these radio button variable names to start with target so it's more clear while you're here?

Reply:
Done.  It's also odd that the source buttons are member-scoped, but the target buttons are local-scoped.  Do you know if there's a room for that?  I'm not spotting one in the code (aside from some changeable order-of-operations)